nntrainer/include/flatten_layer.h

```diff
@@ -86,6 +87,14 @@ class FlattenLayer : public Layer {
    */
   Tensor backwarding(Tensor in, int iteration);
 
+  /**
+   * @brief     set Property of layer
+   * @param[in] values values of property
+   * @retval #ML_ERROR_NONE Successful.
+   * @retval #ML_ERROR_INVALID_PARAMETER invalid parameter.
+   */
+  int setProperty(std::vector<std::string> values) { return ML_ERROR_NONE; };
```

Comment:
How about returning `ML_ERROR_NOT_SUPPORTED` leading to an error when trying to set property, or giving out a warning that this property is being ignored?

Reply:
fixed. Thanks.